setuptools_rust/setuptools_ext.py

```diff
@@ -219,6 +219,19 @@ def get_tag(self):
         dist.cmdclass["bdist_wheel"] = bdist_wheel_rust_extension
 
 
+def patch_disutils_build():
```

Comment:
It would be nice in the source to refer to the upstream `distutils` issue, so that if it's ever merged there's a record of when we might be able to remove this.